setup.py

```diff
@@ -104,6 +104,10 @@
         "fire",
         "hydra-core==1.3.2",
         "omegaconf==2.3.0",
+        "wandb~=0.17.5",
+        "frozendict>=2.4.4",
+        "lru-dict",
+        "mip",
```

Comment:
`mip` installation fails in the pytorch docker container: https://github.com/NVIDIA/TensorRT-Model-Optimizer/actions/runs/19281849124/job/55134742589?pr=539#step:7:591